### PR TITLE
macOS/iOS recovery library bindings

### PR DESCRIPTION
**Pull request by Foxushka**

### CODE REVIEW

File: chameleonultragui/pubspec.lock [hunk 1]
```diff
@@ -710,6 +710,14 @@ packages:
...
     source: hosted
     version: "2.1.4"
+  web:
+    dependency: transitive
+    description:
+      name: web
+      sha256: dc8ccd225a2005c1be616fe02951e2e342092edf968cf0844220383757ef8f10
+      url: "https://pub.dev"
+    source: hosted
+    version: "0.1.4-beta"
```
Comment (GameTec-live) [inline chameleonultragui/pubspec.lock:720]: why is there a added beta web dependency

File: chameleonultragui/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -510,14 +510,14 @@
...
-				DEVELOPMENT_TEAM = 25CH5SQ4CU;
+				DEVELOPMENT_TEAM = P2523ZA3C9;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = Runner/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = net.thomascannon.chameleonultragui;
+				PRODUCT_BUNDLE_IDENTIFIER = dev.chumleon.internal.chameleonultragui;
```
Comment (GameTec-live) [inline chameleonultragui/ios/Runner.xcodeproj/project.pbxproj:520]: dev.chameleon.internal.chameleonultragui

File: chameleonultragui/ios/Runner.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -487,14 +487,14 @@
...
```
Comment (GameTec-live) [inline chameleonultragui/ios/Runner.xcodeproj/project.pbxproj:497]: same here

File: chameleonultragui/ios/Runner.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -358,14 +358,14 @@
...
```
Comment (GameTec-live) [inline chameleonultragui/ios/Runner.xcodeproj/project.pbxproj:368]: and here

**Review by GameTec-live — CHANGES_REQUESTED** [review]

File: chameleonultragui/pubspec.lock (hunk 1 above)
Comment (Foxushka) [reply chameleonultragui/pubspec.lock:720]: idk, who cares

File: chameleonultragui/ios/Runner.xcodeproj/project.pbxproj (hunk 2 above)
Comment (Foxushka) [reply chameleonultragui/ios/Runner.xcodeproj/project.pbxproj:520]: internal, will change when we will publish to App Store (to make sure no one steals our ID)

**Review by GameTec-live — APPROVED** [review]

**Pull request merged**